src/product-card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.body__card {
+  width: 200px;
+}
```

Comment:
Body is a different BEM block so this shouldn't be here

Reply:
ok. done